test/inductor/test_torchinductor.py

```diff
@@ -535,10 +535,10 @@ def populate(cls):
 class TestIndexingSimplification(TorchTestCase):
     def test_indexing_simplification(self):
         sizevars = SizeVarAllocator()
-        i0 = sympy.Symbol("i0")
-        i1 = sympy.Symbol("i1")
-        i2 = sympy.Symbol("i2")
-        r3 = sympy.Symbol("r3")
+        i0 = sympy.Symbol("i0", integer=True)
+        i1 = sympy.Symbol("i1", integer=True)
+        i2 = sympy.Symbol("i2", integer=True)
+        r3 = sympy.Symbol("r3", integer=True)
```

Comment:
Is this cool? I've changed it after seeing:

```
AssertionError: Object comparison failed: 128*i2 + i1//1 + 64*ModularIndexing(r3, 1, 2) != i1 + 128*i2 + 64*ModularIndexing(r3, 1, 2)
```

Since the only way to simplify on 1 right away is for i1 to be an integer.

Reply:
This LGTM!